plan/new_physical_plan_builder.go

```diff
@@ -311,24 +303,10 @@ func (p *LogicalJoin) convert2SemiJoin(prop *requiredProp) (task, error) {
 		Anti:            p.anti,
 	}.init(p.allocator, p.ctx)
 	semiJoin.SetSchema(p.schema)
-	lTask, err := lChild.convert2NewPhysicalPlan(&requiredProp{taskTp: rootTaskType})
-	if err != nil {
-		return nil, errors.Trace(err)
-	}
-	rTask, err := rChild.convert2NewPhysicalPlan(&requiredProp{taskTp: rootTaskType})
-	if err != nil {
-		return nil, errors.Trace(err)
-	}
-	task := semiJoin.attach2Task(lTask, rTask)
-	// Because hash join is executed by multiple goroutines, it will not propagate physical property any more.
-	// TODO: We will consider the problem of property again for parallel execution.
```

Comment:
This TODO comment will be added again when I begin to refactor the logic of cost calculation.